src/main/java/seedu/bitebuddy/logic/commands/CompareCommand.java

```diff
@@ -0,0 +1,124 @@
+package seedu.bitebuddy.logic.commands;
+
+import static seedu.bitebuddy.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import seedu.bitebuddy.commons.core.index.Index;
+import seedu.bitebuddy.commons.util.ToStringBuilder;
+import seedu.bitebuddy.logic.Messages;
+import seedu.bitebuddy.logic.commands.exceptions.CommandException;
+import seedu.bitebuddy.model.Model;
+import seedu.bitebuddy.model.foodplace.Foodplace;
+
+/**
+ * Compares two foodplaces in BiteBuddy.
+ */
+public class CompareCommand extends Command {
+
+    public static final String COMMAND_WORD = "compare";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Compares the two foodplaces identified "
+            + "by the index numbers used in the last foodplace listing. "
+            + "Parameters: INDEX1 and INDEX2 (must be positive integers) "
+            + "compare [INDEX1] [INDEX2]\n"
```

Comment:
Looks like a long line might be returned at once in the command box, maybe using more line breaks might look neater here!

```suggestion
            + "by the index numbers used in the last foodplace listing.\n"
            + "Parameters: INDEX1 and INDEX2 (must be positive integers)\n"
            + "compare [INDEX1] [INDEX2]\n"
```